src/ApiMethodDocumentation.js

```diff
@@ -402,7 +407,7 @@ export class ApiMethodDocumentation extends AmfHelperMixin(LitElement) {
   }
 
   get _titleHidden() {
-    if (!this.noTryIt) {
+    if (this._showTryIt()) {
```

Comment:
Since the component has `noTryIt` property that is designed to manipulate of the state of the request editor, I don't think this component is the right place to put this automation. You can create a helper library here to make the required computations and use this library in the endpoint documentation or the API documentation elements and control the component state through attributes from there.

Reply:
Oh, right. That makes sense. I'll put this logic in `api-documentation`